redirects.txt

```diff
@@ -140,4 +140,6 @@ helpdesk/invoice_time.rst helpdesk/timesheet_and_invoice/invoice_time.rst
 helpdesk/reinvoice_from_project.rst helpdesk/timesheet_and_invoice/reinvoice_from_project.rst    # (#565)
 
 planning/duplicate_a_planning.rst planning/overview/duplicate_a_planning.rst    # (#567)
-planning/send_planned_shifts.rst planning/overview/send_planned_shifts.rst      # (#567)
\ No newline at end of file
+planning/send_planned_shifts.rst planning/overview/send_planned_shifts.rst      # (#567)
+
+expense/expense.rst expenses/expenses.rst
```

Comment:
I see several problems here:
1. Pretty much useless change in a stable version.
2. The content isn't even updated. If you really need to change this, make it part of an overhaul of the page.
3. "expenses/expenses" sounds awfully like the kind of URI that will be updated again as soon as a new Expenses page is added.